scripts/cli/tasks/closeMilestone.ts

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios';
```

Comment:
It's been **years** since I've written any JavaScript, happy to receive feedback on this.